extensions/wrapper/clients/typescript-client/README.md

```diff
@@ -46,6 +48,21 @@ let kpiData: KpiResult = await edcClient.useCaseApi.getKpis();
 A minimal example project using the typescript API client can be found
 [here](https://github.com/sovity/edc-extensions/tree/main/extensions/wrapper/clients/typescript-client-example).
 
+### Example Using OAuth2 Client Credentials
+
+```typescript
+const edcClient: EdcClient = buildEdcClient({
+    managementApiUrl: 'http://localhost:11002/api/management/v2',
+    oAuth2ClientCredentials: {
```

Comment:
Comment: do you think we'd profit from simplifying the name to `clientCredentials`?